cmd/habitat/proxy/p2p.go

```diff
@@ -0,0 +1,50 @@
+package proxy
+
+import (
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"time"
+
+	gostream "github.com/libp2p/go-libp2p-gostream"
+	p2phttp "github.com/libp2p/go-libp2p-http"
```

Comment:
curious why these aren't like `go-libp2p/http`

Reply:
its because they have JS and Rust repos under that github org as well. That being said, IPFS/LibP2p package naming and management is a huge mess